protocol/src/main/java/io/camunda/zeebe/protocol/record/value/BpmnElementType.java

```diff
@@ -55,7 +55,12 @@ public enum BpmnElementType {
 
   BUSINESS_RULE_TASK("businessRuleTask"),
   SCRIPT_TASK("scriptTask"),
-  SEND_TASK("sendTask");
+  SEND_TASK("sendTask"),
+
+  // Data
+  DATA_OBJECT(null),
+  DATA_OBJECT_REFERENCE(null),
+  DATA_STORE_REFERENCE(null);
```

Comment:
:x: We can remove this addition. 

The `BpmnElementType` is only used for `ExecutableFlowElement` objects. Since we aim to skip the `BpmnElementType` assignment for non-executable elements in `[FlowElementInstantiationTransformer.java](https://github.com/camunda/zeebe/pull/10845/files#diff-d920f8b5e298a7ea3d2797fc3a85f43ff1ca7cfc9f57df5a0ba1691e5cdf993c), we don't need to define enums for the non-executable elements.